src/main/java/step5/RacingGame.java

```diff
@@ -0,0 +1,61 @@
+package step5;
+
+import step5.View.InputView;
+import step5.View.OutputView;
+import step5.domain.Car;
+import step5.domain.Cars;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+    private final List<Car> cars;
+    private List<Car> carList = new ArrayList<>();
+    private List<Car> roundWinnerList;
+    private Cars eachCar;
```

Comment:
4개의 변수를 하나로 관리해볼 수 있을 것 같아요 :)
```suggestion
    private final Cars cars;
```